search-service/src/main/resources/db/migration/V0_43__add_csr.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE public.context_source_registration
```

Comment:
be careful, when in multi-tenant, each tenant is in its own schema (not `public`). so don't specify a schema here.